mysql-test/suite/galera/t/galera_parallel_slave.test

```diff
@@ -0,0 +1,39 @@
+
+
+# Start up mysqld with data directory created before the patch
+# Relay log contains data causing deadlock
+# Server without patch will deadlock immediately after startup
+
+--echo # Copy and unzip the datadir
+--copy_file $MYSQLTEST_VARDIR/std_data/pxc4116.zip $MYSQL_TMP_DIR/pxc4116.zip
+--file_exists $MYSQL_TMP_DIR/pxc4116.zip
+--exec unzip -qo $MYSQL_TMP_DIR/pxc4116.zip -d $MYSQL_TMP_DIR/pxc4116z
+
+--echo # Set different datadir path
+--let $MYSQLD_DATADIR= $MYSQL_TMP_DIR/pxc4116z/data2
+--let $MYSQLD_LOG= $MYSQLTEST_VARDIR/log/pxc4116.log
+
+
+use mysql;
```

Comment:
Why do we need this?

Reply:
The data.zip I attached doesn't have a "test" database in it. I thought it was easier to use a database that exists in both than to recreate the zip again.